scripts/lib/mkcloud-common.sh

```diff
@@ -98,6 +101,21 @@ function is_suse
     grep -qi suse /etc/*release
 }
 
+function is_onhost
+{
+    [[ $BASH_SOURCE =~ mkcloud$ ]]
```

Comment:
This does not work, as seen in https://ci.suse.de/job/openstack-mkcloud/38420/console:

```
19:29:21 +(mkcloud-common.sh:106) is_onhost(): [[ /root/scripts/lib/mkcloud-common.sh =~ mkcloud$ ]]
```